docs/implementation.md

```diff
@@ -0,0 +1,210 @@
+## Implementing Links
+
+Before implementing your own Link, be sure to read the [usage](usage.md) document.
+
+Links implement a `request` method that takes an operation and returns the results in an Observable.
+The apollo-link's `Observable` follows the ECMAScript [proposal](https://github.com/tc39/proposal-observable#api) with some extensions provided by [zen-observable](https://github.com/zenparsing/zen-observable#api).
+
+### Creating a Simple Http Link
+
+All links extend the base class, `ApolloLink`, which contains the methods for composition as well as the signature of request.
+The most basic link is a stand-alone Http Link, which implements the `request` method.
+
+```js
+class HttpLink extends ApolloLink {
+
+  constructor(uri){
+    super();
+    this.apolloFetch = createApolloFetch({ uri });
+  }
+
+  request(operation) {
+    return new Observable(observer => {
```

Comment:
Ooh, that's interesting. Maybe a future version should have ways for that to be exposed to the user.